index.html

```diff
@@ -258,6 +262,60 @@ <h3>Video Content Hints</h3>
       </p></td></tr>
     </tbody></table>
   </section>
+  <section>
+    <h2>Behaviors of other components based on content-hint</h2>
+    <section>
+      <h3>Behavior of a MediaStreamTrack</h3>
+      <p>A MediaStreamTrack for which a contentHint attribute has been
+      set MUST set the following default settings for constrainable
+        values:</p>
+        <ul>
+          <li>For an audio track with the value "music",
+            echoCancellation, autoGainControl and noiseSuppression
+            default to "false".</li>
+          <li>For an audio track with the value "speech",
+            echoCancellation and autoGainControl default to
+            "true".</li>
+        </ul>
+        <p>There is no change to the MediaTrakcCapabilities based on
+          this value; constraints may still cause these settings to
+          have any legal value.
+        </p>
+    </section>
+    <section>
+      <h3>Behavior of an RTCPeerConnection</h3>
+      <p>An RTCRtpSender transmitting a MediaStreamTrack for which a
+        contentHint attribute has been
+        set MUST perform the following actions:
+      </p>
+      <ul>
+        <li>For a video track with the attribute value "motion", if
+        RTCDegradationPreference is set to "balanced" (the default),
+        act as if it had been set to "maintain-framerate". 
+        </li>
+        <li>For a video track with the attribute value "detail", if
+        RTCDegradationPreference is set to "balanced", act as if it
+          had been set to "maintain-resolution".
+        </li>
+        <li>For a video track with the attribute value "text", if
+        RTCDegradationPreference is set to "balanced", act as if it
+          had been set to "maintain-resolution". In addition, if the
+        encoding codec is AV1, activate encoding tools for "text" mode.
```

Comment:
I have similar questions here about when observable effects happen (are there any)?

In this case the effect is in a different (sink) object, and there may be more than one sink affected.

Reply:
I suspect that adjusting codec parameters can take time - that some codecs in some settings would do it only at a group-of-frames boundary, for instance. I find it challenging to provide a guarantee of when things will have happened.
What's the right way to say this? "Fire a task that modifies the codec settings at a time appropriate for the codec"?

Reply:
> What's the right way to say this?

Probably *"in parallel"*, though if there are no observable effects... We could specify scope perhaps. Something like:

"For each RTCRtpSender *sender* in this document, whose *sender*.[[SenderTrack]] is *track* and whose associated RTCRtpTransceiver's [[Stopped]] slot is `false`, take steps in parallel to modify *sender*'s codec settings to *newValue* at a time appropriate for its codec"

? Do we need reciprocal mentions of hints in the `degradationPreference` steps as well?

Reply:
This might work, but I still have the question of *when* this determination is to be made.

From your comments it sounds like the intent is for this determination to be made continuously. If so, we should say that. It might be helpful to add a note to implementers about when this might be. I.e.:

1. content hint set or cleared on _track_: update all non-stopped bandwidth-constrained RTCRtpSenders wo/degradationPreference sending _track_ , regardless of source behind _track_.
2. await sender.replaceTrack(newTrackWithDifferentHints)
3. await sender.setParameters(parametersWithDifferentDegradationPreference)

Should we also note that audio constraints do not override content hint in this case, or is that clear?